docs/src/main/sphinx/installation/cli.rst

```diff
@@ -287,3 +292,129 @@ To view debug information, including the stack trace for failures, use the
     at java.base/java.lang.Thread.run(Thread.java:834)
     select count(*) from tpch.tiny.nations
 
+.. _cli-options:
+
+Options
+-------
+
+The following table lists all supported command line options for the Trino CLI
+in alphabetical order:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--access-token``
+    - Access token for :ref:`cli-jwt-auth`.
+  * - ``--catalog``
+    - Default catalog.
+  * - ``--client-info``
+    - Extra information about client making query.
+  * - ``--client-request-timeout``
+    - Client request timeout (default: 2m).
+  * - ``--client-tags``
+    - Client tags.
+  * - ``--debug``
+    - Enable debug information.
+  * - ``--disable-compression``
+    - Disable compression of query results.
+  * - ``--execute``
+    - Execute specified statements and exit.
+  * - ``--external-authentication``
+    - Enable external authentication for :ref:`cli-external-sso-auth`.
+  * - ``--external-authentication-redirect-handler``
+    - External authentication redirect handlers: DESKTOP_OPEN, SYSTEM_OPEN,
```

Comment:
Set the external redirect handler for external authentication. Valid options are .. . 

Also make it a link to the section and we might need to add more info there.